.eslintrc

```diff
@@ -32,7 +32,8 @@
     "semi": [1, "always"],
     "no-trailing-spaces": 0,
     "eol-last": 0,
-    "no-unused-vars": 0,
+    "max-len": [1, {"code": 80, "tabWidth": 2, "ignoreUrls": true}],
```

Comment:
Would be good to make this change to your .eslintrc - it will protect you from bugs and make your code look a lot better. -Harold